modules/ContextUtils.js

```diff
@@ -105,6 +111,12 @@ export function connectToContext(WrappedComponent, name, contextType = PropTypes
       this.unsubscribe = null
     },
 
+    handleContextUpdate(eventIndex) {
```

Comment:
No particular reason. Felt like I might as well take advantage of the autobinding in `React.createClass`.